src/Symfony/Component/Form/Tests/ChoiceList/Factory/DefaultChoiceListFactoryTest.php

```diff
@@ -443,7 +443,7 @@ public function testCreateViewFlatGroupByEmpty()
             array($this->obj2, $this->obj3),
             null, // label
             null, // index
-            array() // ignored
+            null // ignored
```

Comment:
legit change? if it's a bug fix, it should be fixed on lower branches I suppose?

Reply:
I think, yes. This parameter is `groupBy`, it can be `null` or `callable` only. I will do a separate PR later.

Reply:
@nicolas-grekas , done in https://github.com/symfony/symfony/pull/29769. Once merged, I will rebase this PR.